### DISCUSSION TITLE
Facing issue with How to apply Transform Perspective blog https://blog.roboflow.com/estimate-speed-computer-vision/ on my video

### DISCUSSION THREAD

**Discussion by hammad2008** [roboflow/supervision#1036] [Q&A]
Hello. I hope you are doing well. I am facing an issue with How to apply Transform Perspective [How to Estimate Speed with Computer Vision](https://blog.roboflow.com/estimate-speed-computer-vision/) on my video.

import cv2
file_path = “/content/pexels-zero-:heart:-��-12914057 (2160p).mp4” # change to your own video path
vid = cv2.VideoCapture(file_path)
height = vid.get(cv2.CAP_PROP_FRAME_HEIGHT)
width = vid.get(cv2.CAP_PROP_FRAME_WIDTH)
print(height,width)

The size of input frame of video is 2160.0 3840.0

The Polygon is draw like this which is incorrect . i want to change it please help me in doing that.
image

![image](https://github.com/roboflow/supervision/assets/75312856/a8a173b7-b629-4d1c-9af2-43885dbd4a2b)

**Comment by hammad2008** [comment]
@skylargivens @sberan @hansent @tonylampada  Can any of you help me solving this problem please.

**Comment by SkalskiP** [comment]
Hi @hammad2008 👋🏻 Did you updated `SOURCE` and `TARGET` values [here](https://github.com/roboflow/supervision/blob/495f9a90030e8eded73123bece57a37044e0f93c/examples/speed_estimation/ultralytics_example.py#L10)?

**Reply by hammad2008** [reply]
@SkalskiP No, I have not change it thats why i post a question how to change it, The size of input frame of video is 2160.0,3840.0

**Reply by hammad2008** [reply]
@SkalskiP @skylargivens  Can you help me solving this?

**Reply by SkalskiP** [reply]
@hammad2008 I explained the whole process in: https://www.youtube.com/watch?v=uWP6UjDeZvY YT tutorial.

**Reply by hammad2008** [reply]
@SkalskiP i go through the video but i am unable to create a correct box. Please help me to do that. My dimension are mentioned above.

**Reply by hammad2008** [reply]
@SkalskiP here the linkfor my video https://drive.google.com/file/d/1u6tB5F-ADWZt9-u4BCAQXEs694aN3fTV/view?usp=sharing .please help me in that
